console/console-server/src/main/resources/iot.schema.graphql

```diff
@@ -73,11 +74,20 @@ type IotEndpoint {
 type Device {
   deviceId: String!
   enabled: Boolean!
-  viaGateway: Boolean!
-  jsonData: String! #The Json representation of this device.
+  via: [String!]
+  viaGroups: [String!]
+  memberOf: [String!]
+  status: Device_status!
+  ext: String! #The Json representation of the ext field for this device.
   credentials: String! #A Json array with the devices credentials.
 }
 
+type Device_status {
+  created: String!
+  updated: String!
+  last_user: String!
```

Comment:
`lastSeen` field is missing, unable to get it in the device query.

Reply:
good catch ! It's updated